src/com/reader/TXTReader.java

```diff
@@ -0,0 +1,57 @@
+package com.reader;
+
+import org.apache.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Class that contains the txt file reader methods
+ *
+ * @author Jose Cabrera
+ */
+public class TXTReader extends AbstractReader {
+
+    public TXTReader(String filePath) {
+        super(filePath);
+    }
+
+    /**
+     * Method to read a txt file verifying that the file exist
+     *
+     * @return parsedText
+     */
+    @Override
+    public String getContentFile() {
+        String parsedText = "";
+        File file = new File(filePath);
+        if (file.exists()) {
+            parsedText = read(file).replace(",", "");
```

Comment:
Why the TXT and the CSV are doing the same, if that is the case why it is not defined in the parent class